connectivity/mqtt/duplex_utils.c

```diff
@@ -104,10 +104,11 @@ status_t gossip_api_channels_set(mosq_config_t *channel_cfg, char *host, char *r
   char *sub_topic = NULL;
   int sub_topic_len, api_name_len;
   int root_path_len = strlen(root_path);
-  char *api_names[API_NUM] = {"address",          "tag/hashes", "tag/object", "transaction/object",
-                              "transaction/send", "tips/all",   "tips/pair"};
+  char *api_names[] = {"address", "tag/hashes", "tag/object", "transaction/object", "tryte",
+                              "transaction/single", "transaction/send", "tips/all", "tips/pair"};
 
-  for (int i = 0; i < API_NUM; i++) {
+  int api_num = sizeof(api_names)/sizeof(api_names[0]);
```

Comment:
Shall we introduce macro `ARRAY_SIZE` as Linux kernel does?
Reference: https://elixir.bootlin.com/linux/v5.5/source/include/linux/kernel.h#L47

Reply:
Yeah, `ARRAY_SIZE` came to my mind before making the modification, but I'm not sure whether should we introduce a macro that is only used once or not.

Reply:
A few other place can use `ARRAY_SIZE` as well, such as [`cli_info.h`](https://github.com/DLTcollab/tangle-accelerator/blob/develop/accelerator/cli_info.h#L89)